### ISSUE TITLE
Add Enum to the GameAsset, AssetTexture and other Asset nodes

### ISSUE THREAD

**Issue by antongit** [vvvv/VL.Stride#343]
At the moment one have to type an URL.

![image](https://user-images.githubusercontent.com/1189037/99741197-d466c080-2ad0-11eb-9b7f-fe7aaa819a72.png)

Which textures are included by default? 
How to know their URLs?

**Comment by tebjan** [comment]
yes, that would be the best. at the moment we only have the list here: https://github.com/vvvv/VL.Stride/tree/develop/packages/VL.Stride.Runtime/src/Assets

the URL is `FolderName/filename` but this contains many old files that are not needed anymore. see also #250
